### ISSUE TITLE
Data missing from split-out lesson repos

### ISSUE THREAD

**Issue by hlapp** [datacarpentry/planning#10]
_From @hlapp on March 28, 2015 16:48_

The question about where to put (and thus from where to get within a lesson) the data set needs to be re-determined. Originally, there was to be a `data/` directory in the main lesson repo, but that doesn't seem to have come along when splitting out lessons, and so now it's not clear where the data goes.

_Copied from original issue: datacarpentry/datacarpentry#141_

**Comment by hlapp** [comment]
_From @ethanwhite on March 28, 2015 17:42_

The data is now available externally via figshare:
http://figshare.com/articles/Portal_Project_Teaching_Database/1314459

I think we should move all lessons to accessing the data directly from there.

**Comment by hlapp** [comment]
> The data is now available externally via figshare:
> http://figshare.com/articles/Portal_Project_Teaching_Database/1314459

Good point. Should lessons start with a point-and-click download from there, or with a series of copy&paste `curl` commands?

**Comment by hlapp** [comment]
_From @ethanwhite on March 28, 2015 18:54_

Given that this is novices I think point-and-click downloads are the way to go unless the lesson is specifically dealing with automating this sort of thing.

**Comment by tracykteal** [comment]
I agree point and click. This is maybe better than the direct links to the csv files, which the automatically downloads it, as it is now. 

Also, would it be possible to include 'messy' data in this figshare directory?

**Comment by ethanwhite** [comment]
> Also, would it be possible to include 'messy' data in this figshare directory? 

Yep, as mentioned just about simultaneously in #3 :), let's discuss (in #3) what we want in a messy dataset and then @skmorgane and I will get a `_messy` version added to figshare when we get the chance.
